linta-android/src/main/java/com/swvl/lint/utils/ColorUtils.kt

```diff
@@ -40,7 +40,7 @@ object ColorUtils {
      * @return the 8-digit hexadecimal format of a color of any length.
      */
     fun get8DigitColorFormat(hexColor: String): String {
-        val newColor = hexColor.toUpperCase(Locale.US)
+        val newColor = hexColor.uppercase(Locale.US)
```

Comment:
No need to set a specific locale anymore with this extension function.

```suggestion
        val newColor = hexColor.uppercase()
```